src/main/java/com/icthh/xm/ms/otp/config/SecurityConfiguration.java

```diff
@@ -84,8 +84,8 @@ public RestTemplate loadBalancedRestTemplate(RestTemplateCustomizer customizer)
     @Bean
     @Qualifier("internalRestTemplate")
     public RestTemplate internalRestTemplate(RestTemplateCustomizer customizer,
-                                             @Value("${jhipster.security.client-authorization.client-id}") String clientId,
-                                             @Value("${jhipster.security.client-authorization.client-secret}") String clientSecret) {
+                                             @Value("${server.security.client-authorization.client-id}") String clientId,
```

Comment:
why this configuraton is changed?

Reply:
ServerProperties is used instead of JHipsterProperties

Reply:
fixed